src/mpyl/project.py

```diff
@@ -269,6 +269,8 @@ def from_config(values: dict):
 @dataclass(frozen=True)
 class Kubernetes:
     port_mappings: dict[int, int]
+    # once mpyl is the main driver for deployments, we can move kubernetes.secrets to properties.secrets
+    secrets: list[dict[str, str]]
```

Comment:
What kind of secrets are these? Can you link a project yaml example? Because we have now sealed secrets, but also "normal" k8s secrets (done in #165), and now this is a third one?

Reply:
https://github.com/Vandebron/scripting/blob/7256aa60b4afef60dadf227b323ef2c55a19a4db/python/production_correction/deployment/project.yml

This is how it looks for example

Reply:
Oof that's indeed a third one.. @Jorg88

Reply:
They get reference in the user-code-deployment of dagster.

I would love to make it fit into the strcutre of #165

Reply:
those secrets get set as `envSecret` in the user-code-deployments (see https://github.com/dagster-io/dagster/blob/3d0884e617e58bc1e97f9f40c4096e7189bfb678/helm/dagster/values.yaml#L340)

Reply:
Ah indeed I see from git blame, this secret's mechanism is added by Pieter for deploying Dagster. In that case, we should try to combine it with the approach of #165, or move it to the dagster config if it's possible

Reply:
moving them to dagster seems feasible,
i think the future idea was to marry sealed-secrets and dagster

Reply:
That'd be the best!